build.rs

```diff
@@ -382,7 +383,6 @@ mod c {
             sources.extend(&[
                 ("__absvti2", "absvti2.c"),
                 ("__addvti3", "addvti3.c"),
-                ("__clzti2", "clzti2.c"),
```

Comment:
I think you could also remove `__clzdi2` and `__clzsi2` on lines 349-350 now that these aren't broken. Possibly https://github.com/rust-lang/compiler-builtins/blob/2bf0425e3f4853935b1ea1d2865edf5d3bca3074/build.rs#L577 too if our implementation works there.

Reply:
I wasn't sure what's the policy on these. Is it "only apply C replacements for broken/missing functions"?

Reply:
That is accurate, part of the goal of this crate is to be able to target some platforms without needing a C toolchain. I think that usually once something gets ported over and tested, it can be removed from the C sources lists.

Reply:
Done.

Aarch64 CI is down because of new rustc I guess (0/1 asm labels are now off limits instead of just discouraged). This particular thing about numeric labels amazes me so much... This is due to llvm bug, but instead of dealing with that people decided to document it in rust book (so I guess it is a feature now) and make it a compile-time error.

Reply:
I don't think it's an easy llvm bug to fix. That lint should almost certainly be x86-only though where LLVM was already throwing an error, I brought it up.

You can add

```
#![allow(binary_asm_labels)] // aarch64 has no problems with binary labels
```

to the top of `src/aarch64_linux.rs` just to get that to pass.